public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -674,7 +674,7 @@ export class PrometheusDatasource extends DataSourceWithBackend<PromQuery, PromO
           data: {
             from: (this.getPrometheusTime(options.range.from, false) * 1000).toString(),
             to: (this.getPrometheusTime(options.range.to, true) * 1000).toString(),
-            queries: [queryModel],
+            queries: [this.applyTemplateVariables(queryModel, {})],
```

Comment:
Maybe the second parameter should be filled in `options.dashboard.getVariables()`.
I think empty `scopedVars` will not make any thing change.